library/core/src/ptr/mod.rs

```diff
@@ -742,8 +742,9 @@ pub const unsafe fn read<T>(src: *const T) -> T {
 ///
 /// ## On `packed` structs
 ///
-/// It is currently impossible to create raw pointers to unaligned fields
-/// of a packed struct.
+/// You _can_ use this function to write to pointers to unaligned fields of a
+/// packed struct. This was previously not possible due a limitation in
+/// creating the pointers itself (the function was fine).
 ///
```

Comment:
I'm not sure this paragraph is very helpful without knowing the history here. Maybe we should just leave this paragraph out, since it's already covered by what's below.
```suggestion
```